widget/entry.go

```diff
@@ -426,25 +426,41 @@ func (e *Entry) Focused() bool {
 func (e *Entry) cursorColAt(text []rune, pos fyne.Position) int {
 	for i := 0; i < len(text); i++ {
 		str := string(text[0 : i+1])
-		wid := textMinSize(str, theme.TextSize(), e.textStyle()).Width
+		wid := textMinSize(str, theme.TextSize(), e.textStyle()).Width + theme.Padding()
```

Comment:
This looks like something strange, why do we need to add a padding amount?

Reply:
The text is rendered with padding so in order to compare text width against click position we need to take padding into account in the width calculation.

Reply:
There is something not quite right about this - try selecting the letter “I” and you’ll see it is off somehow. I can look into it more later but for now just adding padding feels like an approximation?

Reply:
I'll send you a recording in gophers